gamestonk_terminal/economy/fred/fred_view.py

```diff
@@ -51,9 +51,73 @@ def notes(series_term: str, num: int):
     print("")
 
 
-def display_series(series: str, start_date: str, raw: bool, export: str):
+def display_fred_series(
+    d_series: Dict, start_date: str, raw: bool = False, export: str = ""
+):
     """Display (multiple) series from https://fred.stlouisfed.org. [Source: FRED]
+    Parameters
+    ----------
+    series : str
+        FRED Series ID from https://fred.stlouisfed.org. For multiple series use: series1,series2,series3
+    start_date : str
+        Starting date (YYYY-MM-DD) of data
+    raw : bool
+        Output only raw data
+    export : str
+        Export data to csv,json,xlsx or png,jpg,pdf,svg file
+    """
+    series_ids = d_series.keys()
+    data = pd.DataFrame()
+
+    for s_id in series_ids:
+        data = pd.concat(
+            [
+                data,
+                pd.DataFrame(
+                    fred_model.get_series_data(s_id, start_date), columns=[s_id]
+                ),
+            ],
+            axis=1,
+        )
+
+    # Try to get everything onto the same 0-10 scale.
+    # To do so, think in scientific notation.  Divide the data by whatever the E would be
+    fig, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
+
+    for s_id, title in d_series.items():
```

Comment:
Nice!

![Screenshot 2021-12-08 at 10 01 02](https://user-images.githubusercontent.com/25267873/145188626-8180f898-6240-4b59-96fc-5b9120204b93.png)

Minor stuff but can we take advantage of K, M, B, and T. It's just much easier to the eye than counting 0s.

You can use this code for that:
```
while abs(num) >= 1000:
            magnitude += 1
            num /= 1000.0
        num_str = int(num) if num.is_integer() else f"{num:.3f}"
        return f"{num_str} {' KMBTP'[magnitude]}".strip()
```

Reply:
I can also look into the units sonce GDP is probably in Trillion, not 10k lol